bikeshed/InputSource.py

```diff
@@ -96,6 +96,11 @@ def cheaplyExists(self, relativePath) -> Optional[bool]:
         """
         return None
 
+    def __getattr__(self, name):
```

Comment:
Seems like a hack based on https://stackoverflow.com/a/60731663/455535

Reply:
Def a hack, but I'm okay with it if properly documented as such. One sec and I'll edit the commit...